api/File.json

```
@@ -239,7 +239,7 @@
               "version_added": true
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": "≤13.1.2"
```

```suggestion
              "version_added": "8"
```

I have tested `'name' in File.prototype` in Safari iOS on BrowserStack, and found that it's present in Safari iOS 8 but wasn't present in 7.